.ci/pipeline/ci.yml

```
@@ -89,6 +89,50 @@ jobs:
     condition: failed()
     continueOnError: true
 
+- job: 'LinuxMakeGNU_OpenBLAS'
+  timeoutInMinutes: 0
+  variables:
+    release.dir: '__release_lnx_gnu'
+    platform.type : 'lnx32e'
+  pool:
+    vmImage: 'ubuntu-22.04'
+  steps:
+  - script: |
+      sudo apt-get update && sudo apt-get install -y gcc-multilib g++-multilib dos2unix tree
+      conda create -n ci-env -q -y -c intel -c conda-forge impi-devel=2021.8.0 openjdk=17.0.3
+    displayName: 'apt-get and conda install'
+  - script: |
+      .ci/scripts/describe_system.sh
+    displayName: 'System info'
+  - script: |
+      .ci/scripts/build.sh --compiler gnu --optimizations avx2 --target daal --backend_config ref --conda-env ci-env
```

sse2 always build by default - forcing this config would build only 1 platform and not both sse2 and avx and should reduce build time
```suggestion
      .ci/scripts/build.sh --compiler gnu --optimizations sse2--target daal --backend_config ref --conda-env ci-env
```